extensions/amp-lightbox/1.0/amp-lightbox.css.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
This should just be a normal CSS file with the `amp-lightbox {}` type of selectors. See the `0.1` for details. You just need to modify config to allow extension CSS. See [here](https://github.com/ampproject/amphtml/blob/6cba3aa84378c86d1f8ed325231ff68ce22b1297/build-system/compile/bundles.config.js#L729) and [here](https://github.com/ampproject/amphtml/blob/6cba3aa84378c86d1f8ed325231ff68ce22b1297/extensions/amp-lightbox/0.1/amp-lightbox.js#L19).